contracts/TopShot.cdc

```diff
@@ -94,6 +94,10 @@ pub contract TopShot: NonFungibleToken {
     // Emitted when a Moment is destroyed
     pub event MomentDestroyed(id: UInt64)
 
+    pub event SubeditionCreated(id: UInt32, name: String, metadata: {String:String})
```

Comment:
Agree with consistent/more specific id.
In the current iteration, the subedition is intended to be reused across many sets/plays. (This will make it much easier to search for all 'diamond' moments if they share an id) -> so subedition created event will not have a set/play as the subedition itself does not have a set/play.

Reply:
```suggestion
    pub event SubeditionCreated(subeditionID: UInt32, name: String, metadata: {String:String})
```